plaidml/op/op.h

```diff
@@ -340,16 +340,11 @@ inline edsl::Tensor minimum(const edsl::Tensor& X, const edsl::Tensor& Y) {
 class mvn {
  public:
   explicit mvn(const plaidml::edsl::Tensor& I)
-      : I_(I),
-        axes_(edsl::None()),
-        normalize_variance_(true),
-        epsilon_(1e-9),
-        across_channels_(true),
-        layout_("NHWC") {}
+      : I_(I), axes_(edsl::None()), normalize_variance_(true), epsilon_(1e-9), across_channels_(true), layout_("") {}
```

Comment:
No need to default to `""` (that's what the default ctor of a `std::string` does already)